test/conformance/kuttl/generate/clusterpolicy/standard/clone/sync/cpol-clone-list-sync-update/README.md

```diff
@@ -0,0 +1,9 @@
+## Description
+
+This test ensures that creation of a multiple target resource created by a ClusterPolicy `generate.cloneList` rule. If it is not generated, the test fails. 
+
+Further it verifies the sync behavior, if the source resource gets changed, the update should be synchronized with the target resource as well.
```

Comment:
Because we already have several generate tests at this point where we're separate create behavior from all others, let's please create a second test case just for the updates. I know it's a few more files, but it will ensure consistency and cleaner separation. Please remember to use the scaffolding dir to pick up the latest version of the test files.

Reply:
This tests verifies the create and update(sync) behaviour, We already have a test to verify only the create, which i am thinking to change that test to cover some other test-case.

Reply:
A test which covers update/delete will have to naturally progress through the creation phase.